edit/edit.js

```diff
@@ -1461,8 +1461,14 @@ function fromMozillaFormat() {
     }, 100);
   });
 
-  function doImport() {
-    const replaceOldStyle = this.name === 'import-replace';
+  function doImport(event) {
+    // parserlib contained in CSSLint-worker.js
+    onDOMscripted(['vendor-overwrites/csslint/csslint-worker.js'])
+      .then(() => doImportWhenReady(event.target));
+  }
+
+  function doImportWhenReady(target) {
+    const replaceOldStyle = target.name === 'import-replace';
```

Comment:
I'm not sure what changing the `name` of the clicked button is supposed to do, but I left it in there...